strongbox-commons/src/main/resources/application.yaml

```diff
@@ -50,6 +52,7 @@ logging:
       total: 1GB
     history: 31
   dir: ${strongbox.vault}/logs
+logging.file: ${logging.dir}/strongbox.log
```

Comment:
this dotted property looks like some kind of workaround :)

can't we use something like `name:` property under regular `logging: ->file:'?

Reply:
we're using standard `logging.file` property of `spring boot < 2.2.X` See also https://github.com/spring-projects/spring-boot/issues/12510 I think that using standard property is better (even with this not so dramatic workaround) than using our custom property. Once we upgrade to 2.2.X we will see if we can use new standard `logging.file.name` property.

Reply:
ok, can you please leave a `TODO` comment to replace it with `logging.file.name` after `2.2.X`?

Reply:
fixed :)